### DISCUSSION TITLE
Option to disable the "one-click unsubscribe" that allows to disable quickly an alias

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#399] [Feature Requests]
This option is initially designed to allow users to quickly disable an alias using the one-click unsubscribe button provided by some email services. 

However, this option seems to create confusion for users who are not familiar with this option. It should be then turned off by default and only users need to explicitly turn it on. 

Moved from https://trello.com/c/GO9hxltw/113-option-to-disable-the-one-click-unsubscribe-that-allows-to-disable-quickly-an-alias

**Comment by denchuku** [comment]
I was also confused for the first 2 weeks until I finally clicked on that link and understood that it's actually meant as a convenience from simple login 😄 
The main problem for me was that my mail client (default mail app on macOS) doesn't show were the unsubscribe link goes to. 

Turning it off per default sounds like a good idea :+1:

**Comment by tylerjmorg** [comment]
This would be great also for filtering out junk email using sieve filters. I used to use this filter before switching to SimpleLogin, but now I can't because I cannot disable the One Click Unsubscribe button.

**Comment by Practicalbutterfly5** [comment]
This option caught me off guard. I think the unsubscribe option by default should simply **preserve the original unsubscribe policy** rather than **disabling alias**, which could have devastating effects if the user used the same alias for multiple websites.

**Reply by dinvlad** [reply]
This is what happened to me as well, multiple times in a row until I understood what's happeneing! Lost an important contact because of it :/

**Reply by nguyenkims** [reply]
It should be fixed with https://github.com/simple-login/app/pull/1652

**Reply by xyxzxyz** [reply]
preservation is broken. It doesn't preserve anymore. It just removes list-unsubscribed from header altogether.
